aws/cloudformation/iam.yml.erb

```diff
@@ -212,7 +212,14 @@ Resources:
             Condition:
               StringEquals:
                 SAML:aud: https://signin.aws.amazon.com/saml
-      ManagedPolicyArns: [!Ref DataAnalystPolicy]
+      ManagedPolicyArns:
+        - !Ref DataAnalystPolicy
+        # TODO: compare what we are giving this role below against these two AWS policies recommended by the docs. Specifically the redshift:*
+        - AmazonRedshiftQueryEditor
+        - AmazonRedshiftQueryEditorV2FullAccess
+    Tags:
+      - Key: sqlworkbench-team
+        Value: red-team
```

Comment:
The "AmazonRedshiftQueryEditorV2FullAccess" policy will grant the ability to view queries from users with the same "sqlworkbench-team" tag.